sharedbuffers/mapped_struct.py

```diff
@@ -2122,6 +2508,7 @@ class mapped_object(object):
         proxied_ndarray: 'n',
         proxied_buffer: 'r',
         proxied_dict: 'M',
+        proxied_frozenset: 'O',
```

Comment:
O looks like Object. Is lowercase `z` taken?

Reply:
I went for 'O' as in 'ordered', but I guess it can be a bit confusing. Lowercase 'z' doesn't appear to be taken.